### PR TITLE
Docs: show configuration reference on builds done at readthedocs.io

### PR DESCRIPTION
**Pull request by clebergnu**
Which doesn't do builds by makefiles, but by using conf.py only. So,
currently, at:

  https://avocado-framework.readthedocs.io/en/latest/config/index.html

The config reference is now being shown, and this is a fix.

Signed-off-by: Cleber Rosa <crosa@redhat.com>

### CODE REVIEW

File: avocado/plugins/config.py [hunk 1]
```diff
@@ -50,22 +50,23 @@ def configure(self, parser):
...
-                LOG_UI.debug(line)
-            LOG_UI.debug("")
-            LOG_UI.debug("* Default: %s", option.get('default'))
-            LOG_UI.debug("* Type: %s", option.get('type'))
-            LOG_UI.debug("")
+                print_function(line)
+            print_function("")
+            print_function("* Default: %s" % option.get('default'))
+            print_function("* Type: %s" % option.get('type'))
+            print_function("")
```
Comment (beraldoleal) [inline avocado/plugins/config.py:69]: I didn't get why this change, but I'm ok with that.

Comment (clebergnu) [reply avocado/plugins/config.py:69]: Do you mean this patch, or this block? I'm curious.

Comment (beraldoleal) [reply avocado/plugins/config.py:69]: Adding a `print_function` argument and replacing instead of using directly the LOG_UI. I'm assuming that is to make it easy to replace the log level at one place instead of multiple lines.

Comment (clebergnu) [reply avocado/plugins/config.py:69]: Because we need to output to a file isntead here:

https://github.com/avocado-framework/avocado/pull/3946/files#diff-1ec61223727a10e5e4c74c68336809fcR40

Comment (beraldoleal) [reply avocado/plugins/config.py:69]: @clebergnu got it. thanks. Need more coffee here.

Comment (clebergnu) [reply avocado/plugins/config.py:69]: > @clebergnu got it. thanks. Need more coffee here.

:coffee:
